jenkins-slaves/ansible/roles/kie/tasks/main.yml

```diff
@@ -158,34 +157,30 @@
   with_items:
    - kiegroup/lienzo-core
    - kiegroup/lienzo-tests
-   - errai/errai
-   - dashbuilder/dashbuilder
+   - kiegroup/droolsjbpm-build-bootstrap
    - kiegroup/kie-soup
    - kiegroup/appformer
-   - kiegroup/droolsjbpm-build-bootstrap
    - kiegroup/droolsjbpm-knowledge
    - kiegroup/drools
    - kiegroup/optaplanner
-   - kiegroup/optaweb-employee-rostering
    - kiegroup/jbpm
-   - kiegroup/jbpm-work-items
```

Comment:
this should not be removed IMHO, see https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/script/repository-list.txt#L20